Sources/Orbit/Components/ListChoice.swift

```diff
@@ -0,0 +1,252 @@
+import SwiftUI
+
+/// Shows one of a selectable list of items with similar structures.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/listchoice/)
+/// - Important: Component expands horizontally to infinity up to a ``Layout/readableMaxWidth``.
+public struct ListChoice: View {
+
+    let title: String
+    let description: String
+    let icon: Icon.Symbol
```

Comment:
For future reusability, we might probably need to use Icon.Content here, as some existing similar custom components already use flags in place of icon for example (and a client may need to provide a custom icon content).